diff -ur a/etc/apache2/sites-available/000-default.conf b/etc/apache2/sites-available/000-default.conf
--- a/etc/apache2/sites-available/000-default.conf	2018-10-31 10:33:10.027230627 +0000
+++ b/etc/apache2/sites-available/000-default.conf	2018-10-31 13:37:26.189541663 +0000
@@ -11,6 +11,8 @@
 	ServerAdmin webmaster@localhost
 	DocumentRoot /var/www/html
 
+	# Shift the URL that everything is served from to /wiki.
+	Alias /wiki /var/www/html
+
 	# Available loglevels: trace8, ..., trace1, debug, info, notice, warn,
 	# error, crit, alert, emerg.
 	# It is also possible to configure the loglevel for particular
